src/main/java/org/fcrepo/camel/FcrepoEndpoint.java

```diff
@@ -34,43 +34,8 @@
 @UriEndpoint(scheme = "fcrepo")
 public class FcrepoEndpoint extends DefaultEndpoint {
 
-    private String baseUrl = "";
-
-    @UriParam
-    private String contentType = null;
-
-    @UriParam
-    private String accept = null;
-
-    @UriParam
-    private String transform = null;
-
-    @UriParam
-    private String authUsername = null;
-
-    @UriParam
-    private String authPassword = null;
-
-    @UriParam
-    private String authHost = null;
-
-    @UriParam
-    private Boolean secure = false;
-
     @UriParam
-    private Boolean tombstone = false;
-
-    @UriParam
-    private Boolean metadata = true;
-
-    @UriParam
-    private Boolean throwExceptionOnFailure = true;
-
-    @UriParam
-    private String preferInclude = null;
-
-    @UriParam
-    private String preferOmit = null;
+    private FcrepoConfiguration configuration;
```

Comment:
It looks like the `@UriParam` annotation may actually be superfluous here.

I looked through the camel source code, and the only time `@UriParams` is used is to decorate the `Configuration` classes, as I'm doing with `FcrepoConfiguration`. As for decorating the configuration field as I'm doing here, no one does that. I'd suggest removing the annotation altogether.